imap_processing/tests/external_test_data_config.py

```diff
@@ -0,0 +1,67 @@
+from imap_processing import imap_module_directory
+
+# ruff: noqa: E501
+EXTERNAL_TEST_DATA = [
+
+    # CoDICE
+    ("imap_codice_l0_raw_20241110_v001.pkts", imap_module_directory / "tests" / "codice" / "data" / "imap_codice_l0_raw_20241110_v001.pkts"),
```

Comment:
Overall, this looks like a nice improvement. I'm wondering if it would be worth taking it a step further to make the tuple for a single file contain `(<s3_filename>, <parent_test_dir_relative_to_imap_module_dir>)`. That would clean up this configuration by removing the `imap_module_directory` as well as the target path filename. The later part would require that the filename remains the same when downloaded.

For this line, it would look like:
```suggestion
    ("imap_codice_l0_raw_20241110_v001.pkts", "tests/codice/data"),
```

Reply:
Good idea, I like it.